ledger/ledger-api-common/src/main/scala/com/digitalasset/platform/server/api/validation/ErrorFactories.scala

```diff
@@ -200,34 +177,48 @@ class ErrorFactories private (errorCodesVersionSwitcher: ErrorCodesVersionSwitch
         addDefiniteAnswerDetails(definiteAnswer, statusBuilder)
         grpcError(statusBuilder.build())
       },
-      // TODO error codes: Revisit this error: This error group is confusing for this generic error as it can be dispatched
-      //                   from call-sites that do not involve command validation (e.g. ApiTransactionService, GrpcHealthService).
-      v2 = LedgerApiErrors.CommandValidation.InvalidArgument
+      v2 = LedgerApiErrors.RequestValidation.InvalidArgument
         .Reject(message)
         .asGrpcError,
     )
 
-  // TODO error codes: Reconcile with com.daml.platform.server.api.validation.ErrorFactories.offsetOutOfRange
-  def offsetOutOfRange_was_invalidArgument(
+  def offsetOutOfRange(
```

Comment:
Returned V2 grpc code changed from `OUT_OF_RANGE` to `FAILED_PRECONDITION`

Reply:
Is that right? This method seems to be kept the same. I think you meant the next one?

Reply:
The underlying returned error changed from `OUT_OF_RANGE` to `FAILED_PRECONDITION`